dataflux_pytorch/tests/test_dataflux_mapstyle_dataset.py

```diff
@@ -337,8 +332,14 @@ def data_format_fn(content):
             retry_config=dataflux_mapstyle_dataset.MODIFIED_RETRY,
         )
 
-    def test_init_sets_user_agent(self):
+    @mock.patch("dataflux_pytorch.dataflux_mapstyle_dataset.dataflux_core")
```

Comment:
It may not matter in this instance, but you don't have to mock the whole package out if you want to be more precise. If you namespace into the module (e.g. `fast_list.ListingController`) you can specifically mock out only that portion of dataflux core without mocking everything else. Generally speaking we want to mock the smallest amount of code necessary.